util/walk.js

```diff
@@ -285,7 +285,23 @@
     c(node.object, st, "Expression");
     if (node.computed) c(node.property, st, "Expression");
   };
-  base.Identifier = base.Literal = base.ExportDeclaration = base.ImportDeclaration = ignore;
+  base.ExportDeclaration = function (node, st, c) {
+    c(node.declaration, st);
+  };
+  base.ImportDeclaration = function (node, st, c) {
+    c(node.source, st, 'Literal');
+    node.specifiers.forEach(function (specifier) {
+      c(specifier, st);
+    });
+  };
+  base.ImportSpecifier = function (node, st, c) {
+    if (node.name) c(node.name, st, 'Identifier');
+    else c(node.id, st, 'Identifier');
```

Comment:
Actually, I see that we don't do that for variable declarations.  I shall remove this code.